### PR TITLE
Update fbcode docs

### PR DESCRIPTION
**Pull request by shagunsodhani**
## Motivation

Update fbcode docs

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes

## Test Plan

NA. Doc changes

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

**Comment by jieru-hu** [comment]
to update internal doc, I think @omry mentioned this needs to be changed on 1.0_branch

**Comment by shagunsodhani** [comment]
> to update internal doc, I think @omry mentioned (in the post comment) that this needs to be changed on 1.0_branch

Ouch! thanks for catching.

**Comment by shagunsodhani** [comment]
I think it should be changed on both the branches no ?

**Comment by jieru-hu** [comment]
> I think it should be changed on both the branches no ?

agreed! :)

File: website/docs/fb/fbcode.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ Hydra includes buck TARGETS you can use in fbcode. In general, if there is TARGE
 1. You can depend on the TARGETS to use Hydra or a plugin.
 2. The TARGETS contains a runnable example.
```
Comment (jieru-hu) [inline website/docs/fb/fbcode.md:23]: should we also include what jack suggested? or maybe the examples are self-explanatory enough

```
 in order to use yamls in fbcode, you need to add them to your TARGETS resources: resources = glob(["**/*.yaml"]),
```

Comment (omry) [reply website/docs/fb/fbcode.md:23]: 1. I think we need links to the code tree for hydra, plugins and examples below.
2. We can also add a direct link to TARGETS of 6_composition, which should contain it. I don't think we need to spell it out, it should overall be familiar to fbcode users.

**Review by omry — CHANGES_REQUESTED** [review]
see comments

File: website/versioned_docs/version-1.0/fb/fbcode.md [hunk 2]
```diff
@@ -22,8 +22,10 @@ Hydra includes buck TARGETS you can use in fbcode. In general, if there is TARGE
...
-targets are under `github/facebookresearch/hydra_1.0`:
-- `:hydra` : Primary target to use in most cases. Includes `hydra_oss` and the `fbcode_defaults`.
+An example TARGET file can be accessed [here](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples/tutorials/basic/your_first_hydra_app/6_composition).
+
+targets are under `github/facebookresearch/hydra`:
+- [:`hydra`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra) : Primary target to use in most cases. Includes `hydra_oss` and the `fbcode_defaults`.
 - `:hydra_oss` : Vanilla Hydra without any Facebook specific targets.
-- `plugins/...`: Plugins that has TARGETS file are runnable in fbcode.
-- `examples/...`: Examples that has a TARGETS file are runnable in `fbcode`.
+- [`plugins`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/plugins): Plugins that have TARGETS file are runnable in fbcode.
```
Comment (omry) [inline website/versioned_docs/version-1.0/fb/fbcode.md:30]:
```suggestion
- [`plugins`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/plugins): Plugins that have a TARGETS file are runnable in fbcode.
```

File: website/versioned_docs/version-1.0/fb/fbcode.md [hunk 3]
```diff
@@ -22,8 +22,10 @@ Hydra includes buck TARGETS you can use in fbcode. In general, if there is TARGE
...
+- [`examples`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples): Examples that have a TARGETS file are runnable in `fbcode`.
```
Comment (omry) [inline website/versioned_docs/version-1.0/fb/fbcode.md:31]: Can you inline an example TARGETS (with a link)? 5_composition can be a gooe one.

File: website/docs/fb/fbcode.md [hunk 4]
```diff
@@ -22,8 +22,10 @@ Hydra includes buck TARGETS you can use in fbcode. In general, if there is TARGE
...
+
 targets are under `github/facebookresearch/hydra`:
-- `:hydra` : Primary target to use in most cases. Includes `hydra_oss` and the `fbcode_defaults`.
+- [:`hydra`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra) : Primary target to use in most cases. Includes `hydra_oss` and the `fbcode_defaults`.
 - `:hydra_oss` : Vanilla Hydra without any Facebook specific targets.
-- `plugins/...`: Plugins that have TARGETS file are runnable in fbcode.
-- `examples/...`: Examples that have a TARGETS file are runnable in `fbcode`. All tutorials (in `examples/tutorials`) are supported.
+- [`plugins`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/plugins): Plugins that have TARGETS file are runnable in fbcode.
+- [`examples`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples): Examples that have a TARGETS file are runnable in `fbcode`.
```
Comment (omry) [inline website/docs/fb/fbcode.md:31]: are there any tutorials without a TARGETS file? why did you change the sentence?

**Review by omry — CHANGES_REQUESTED** [review]

File: website/docs/fb/fbcode.md (hunk 4 above)
Comment (shagunsodhani) [reply website/docs/fb/fbcode.md:31]: I think I saw something but now I cant seem to find it. Will revert this.

File: website/versioned_docs/version-1.0/fb/fbcode.md [hunk 5]
```diff
@@ -22,8 +22,8 @@ Hydra includes buck TARGETS you can use in fbcode. In general, if there is TARGE
...
+- [:`hydra`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra) : Primary target to use in most cases. Includes `hydra_oss` and the `fbcode_defaults`.
 - `:hydra_oss` : Vanilla Hydra without any Facebook specific targets.
-- `plugins/...`: Plugins that has TARGETS file are runnable in fbcode.
-- `examples/...`: Examples that has a TARGETS file are runnable in `fbcode`.
+- [`plugins`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/plugins): Plugins that have a TARGETS file are runnable in fbcode.
+- [`examples`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples): Examples that have a TARGETS file are runnable in `fbcode`. All tutorials (in [`examples/tutorials`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples)) are supported. An example TARGET file can be accessed [here](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples/tutorials/basic/your_first_hydra_app/5_composition).
```
Comment (omry) [inline website/versioned_docs/version-1.0/fb/fbcode.md:29]:
```suggestion
- [`examples`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples): Examples that have a TARGETS file are runnable in `fbcode`. All tutorials (in [`examples/tutorials`](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples)) are supported. An example TARGET file can be found [here](https://www.internalfb.com/intern/diffusion/FBS/browsedir/master/fbcode/github/facebookresearch/hydra/examples/tutorials/basic/your_first_hydra_app/5_composition).
```

**Review by omry — APPROVED** [review]
fix and merge. don't forget to cherry pick to 1.0_branch.
